### ISSUE TITLE
addon couldnt be loaded shader preset support

### ISSUE THREAD

**Issue by daveftw84** [garbear/xbmc#129]
when first booting up on android i get a pop up telling me the addon failed to load, should i be concerened?

**Comment by garbear** [comment]
No, you can safely disable the shader preset add-on in the add-on manager. I'm about to get my NVidia shield working again, I'll address this issue.

**Comment by KOPRajs** [comment]
This is solved upstream, lets close it.
